lib/vimeo.js

```diff
@@ -97,6 +97,11 @@ Vimeo.prototype.request = function vimeo_request (options, callback) {
 		return callback(new Error('You must provide an api path'));
 	}
 
+	// Add leading slash to path if missing
+	if (options.path.indexOf('/') !== 0) {
```

Comment:
I think `.charAt(0) !== '/'` might be faster. indexOf likely looks through a lot of the string if the first character is not a slash.

With that tiny tweak, and an addition to our cla: https://developer.vimeo.com/cla I would love to merge this!